CHANGELOG.asciidoc

```diff
@@ -73,6 +73,31 @@ https://github.com/elastic/beats/compare/v5.0.0-alpha2...master[Check the HEAD d
 
 ////////////////////////////////////////////////////////////
 
+[[release-notes-1.2.3]]
+=== Beats version 1.2.3
+https://github.com/elastic/beats/compare/v1.2.2...v1.2.3[View commits]
+
+==== Bugfixes
+
+*Topbeat*
+
+- Fix high CPU usage when using filtering under Windows. {pull}1598[1598]
+
+*Filebeat*
+
+- Fix rotation issue with ignore_older. {issue}1528[1528]
```

Comment:
Yeah this is difficult because the Changelog is linear but we have branches in code. I sorted by release date but kept the changelog in both if an issue is fixed in both. So people will see that a particular issue is also fixed in 5.0-alpha2. Not sure if there's a better way.